packages/fluent/scss/spreadsheet/_layout.scss

```diff
@@ -203,7 +203,7 @@
     // Cell editor
     .k-spreadsheet-cell-editor {
         padding: 0 3px;
-        line-height: var( --kendo-spreadsheet-line-height, $kendo-spreadsheet-line-height );
+        line-height: 20px;
```

Comment:
we should rather update the variable value instead

Reply:
I kept the variable and changed its value to the hardcoded 20px to be unified with the default value of the spreadsheet row height set dynamically by its JavaScript.